pkg/catalog/outbound_traffic_policies.go

```diff
@@ -301,8 +303,9 @@ func (mc *MeshCatalog) ListMeshServicesForIdentity(identity identity.ServiceIden
 				if backend.Service == upstreamSvc.Name {
 					rootServiceName := kubernetes.GetServiceFromHostname(split.Spec.Service)
 					rootMeshService := service.MeshService{
-						Namespace: split.Namespace,
-						Name:      rootServiceName,
+						Namespace:   split.Namespace,
+						Name:        rootServiceName,
+						ClusterName: "local",
```

Comment:
same here and everywhere for referencing a constant

Reply:
Fixed:
```
	apexServices := mapset.NewSet()
	for _, split := range mc.meshSpec.ListTrafficSplits() {
		svc := service.MeshService{
			Name:        kubernetes.GetServiceFromHostname(split.Spec.Service),
			Namespace:   split.Namespace,
			ClusterDomain: "cluster.local",
		}
```